src/main.c

```diff
@@ -498,6 +498,7 @@ static void handle_sighup(int sock, short flags, void *arg)
 	log_info("got SIGHUP, re-reading config");
 	sd_notify(0, "RELOADING=1");
 	load_config();
+	sbuf_tls_setup();
```

Comment:
I think it would be good to check the exit status here and in case of failure log a message along the lines of "TLS configuration could not be reloaded, keeping old configuration".